gui/test/integration/cypress/integration/HomePage.spec.ts

```diff
@@ -9,4 +9,14 @@ context("Home Page", () => {
     page.totalNodeCount.should("have.text", "1000 nodes");
     page.totalEdgeCount.should("have.text", "52293 relationships");
   });
+
+  it("should show all datasources", () => {
+    page.search.selectedDatasource.should("have.text", "All datasources");
+  });
+
+  it("should show selected datasource", () => {
+    page.search.selectDatasource("gui_test_data");
```

Comment:
These magic strings should be in a separate file. I usually call it TestData, exporting info about the synthetic test data.